plugins/devtools/src/components/DefaultDevToolsPage/DefaultDevToolsPage.tsx

```diff
@@ -37,5 +38,14 @@ export const DefaultDevToolsPage = () => (
         <ConfigContent />
```

Comment:
Note that these two pre-existing ones should be made into extensions themselves as well. We can do it as a standalone effort perhaps, but note that for example the [entity overview page itself](https://github.com/backstage/backstage/blob/39a15fdf4dd1cc53da5dfa1563a69048e56d2d8d/plugins/catalog/src/alpha/entityContents.tsx#L33) is a content.

Making them extensions makes it possible for users to rearrange, enable/disable, group and reconfigure them just like the other ones.

Reply:
Do you want to merge this before or after this one?